readme.md

```diff
@@ -94,7 +104,8 @@ How is the concept of OAuth related to a valet key?
 
 Your Answer:
 ```text
-
+In the case of OAuth, the app specifically requests limited access OAuth Token (valet key) to access the the APIs (enter the car) on the user's behalf.  If
+the user can grant that access, the application can retrieve the unique identifier for establishing the profile.
 ```
```

Comment:
Important thing here is that we are not granting an application full access to, say, our Facebook profile.